packages/connectors/connector-wecom/src/index.ts

```diff
@@ -0,0 +1,199 @@
+/**
+ * The Implementation of OpenID Connect of WeChat Web Open Platform.
+ * https://developers.weixin.qq.com/doc/oplatform/Website_App/WeChat_Login/Wechat_Login.html
+ */
+
+import { assert } from '@silverhand/essentials';
+import { got, HTTPError } from 'got';
+
+import type {
+  GetAuthorizationUri,
+  GetUserInfo,
+  GetConnectorConfig,
+  CreateConnector,
+  SocialConnector,
+} from '@logto/connector-kit';
+import {
+  ConnectorError,
+  ConnectorErrorCodes,
+  validateConfig,
+  ConnectorType,
+  parseJson,
+} from '@logto/connector-kit';
+
+import {
+  authorizationEndpointInside,
+  authorizationEndpointQrcode,
+  accessTokenEndpoint,
+  userInfoEndpoint,
+  scope as defaultScope,
+  defaultMetadata,
+  defaultTimeout,
+  invalidAccessTokenErrcode,
+  invalidAuthCodeErrcode,
+} from './constant.js';
+import type {
+  GetAccessTokenErrorHandler,
+  UserInfoResponseMessageParser,
+  WecomConfig,
+} from './types.js';
+import {
+  wecomConfigGuard,
+  accessTokenResponseGuard,
+  userInfoResponseGuard,
+  authResponseGuard,
+} from './types.js';
+
+const getAuthorizationUri =
+  (getConfig: GetConnectorConfig): GetAuthorizationUri =>
+  async ({ state, redirectUri, headers: { userAgent } }) => {
+    const config = await getConfig(defaultMetadata.id);
+    validateConfig(config, wecomConfigGuard);
+
+    const isWecom: boolean | undefined = userAgent?.toLowerCase().includes('wxwork');
```

Comment:
This logic is not clear for reviewers without context, better to attach some comments or links to documents.

Reply:
Added some comments. 
Here is to switch between different `authorizationEndpoint`